src/main/java/core/Application.java

```diff
@@ -0,0 +1,29 @@
+package core;
+
+import core.lib.Injector;
+import core.model.Manufacturer;
+import core.service.ManufacturerService;
+
+public class Application {
+    private static final Injector injector =
+            Injector.getInstance(Application.class.getPackageName());
+
+    public static void main(String[] args) {
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        Manufacturer mercedes = new Manufacturer("Mercedes", "Germany");
+        Manufacturer volvo = new Manufacturer("Volvo", "Sweden");
+        Manufacturer audi = new Manufacturer("Audi", "Germany");
+        audi.setId(1L);
```

Comment:
do we need this code `audi.setId(1L)`? method `create` should set `id` to the object

Reply:
I want to check how my method update working with same id